migrate.pl

```diff
@@ -397,8 +400,10 @@ sub convertMarkdownToJira {
 sub convertUserMentions {
 	my $textToConvert = shift;
 
-	# Convert user @foo mentions to Jira [~foo] links
-	$textToConvert =~ s/\B\@(\S+)/\[\~$User{$1}\]/g;
+	# Convert user @foo mentions to Jira [~accountid:$personId] links (doesn't seem to work)
+	# $textToConvert =~ s/\B\@(\S+)/\@$1 \[\~acccountid:$JiraUserIds{$User{$1}}\])/g;
+	# Convert user @foo mentions to Jira [@foo|/jira/people/$personId] links
+	$textToConvert =~ s/\B\@([^\s,]+)/\[\@$1\|\/jira\/people\/$JiraUserIds{$User{$1}}\]/g;
```

Comment:
someone online said `[~accountid:$personId]` should work... if you switch to use the v2 endpoints (we are using `latest` for addWorkLog, createIssue, createIssues, createComment). At this stage, didn't try it. Converting it to a link is safer, because even if the user doesn't exist in your map, you still get the original text of the mention.